specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json

```diff
@@ -2663,13 +2663,46 @@
             "type": "string"
           },
           "description": "An array of predefined enum values. Only blockBlob is supported."
+        },
+        "blobIndexMatch": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TagFilter"
+          },
+          "description": "An array of tag based filters, there can be at most 10 tag filters"
         }
       },
       "required": [
         "blobTypes"
       ],
       "description": "Filters limit rule actions to a subset of blobs within the storage account. If multiple filters are defined, a logical AND is performed on all filters. "
     },
+    "TagFilter": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 128,
+          "description": "This is the filter tag name, it can have 1 - 128 characters"
+        },
+        "op": {
+          "type": "string",
+          "description": "This is the comparison operation field, which is used for comparison for object filtering."
```

Comment:
Please add some sample for the op, or give all the possible value, if this is a limited range. Or it will be very difficult for user to set op.

Reply:
For preview only == is supported, and then wider range of operators including >,<,>=,<= will be supported. 
The description will have to be updated each time a support will enabled from server side as we move towards GA, thus the description is left generic.  PM must document this correctly.
The server side has validation and returns proper error Message, stating supported operators

Reply:
Actually, rest doc is just generated from swagger without manual change. So the description will be in the rest doc which will be refence by customer when use this API. 
Same for the comments of SDK in each language.

So please add the supported  operator here, or it will be very difficult for user to use this API.
Besides that, if the operator scope will change in the future, please update it in swagger. Then all docs, SDKs will be updated automatically when regenerate.

Reply:
Besides that, for tools (e.g. PSH, CLI), also need tell user which operatior are supported, and the inforamtion is coming from swagger (feature team).
So keep swagger clear enough  is very important for all following SDK/ Tools/ Docs generation and development.

Reply:
done